meta-lmp-base/recipes-support/bluetooth-attach-conf/bluetooth-attach-conf_0.1.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Bluetooth Serial Attach Configuration"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
+
+RDEPENDS_${PN} += "bluez5 bluetooth-attach"
```

Comment:
I'm not 100% sure we need to create a new recipe just for the configuration, as that can be provided by bluetooth-attach.

This RDEPENDS is a bit backwards, since in reality bluetooth-attach depends on it.

```
cat ./meta-lmp-base/recipes-support/bluetooth-attach/bluetooth-attach/btattach.service
[Unit]
Description=Bluetooth Serial Attach Initialization
ConditionFileNotEmpty=/etc/bluetooth/btattach.conf
After=rc-local.service
```

The condition is already when file is not empty, so we can probably add a default btattach.conf as part of bluetooth-attach that is empty by default, then have a bbappend that only adds the file provided here in a machine specific folder.

Reply:
Ah!  I didn't see the `ConditionFileNotEmpty` change.  Let me get a separate PR to add a default (empty) btattach.conf added to the recipe that can be overriden.

Reply:
https://github.com/foundriesio/meta-lmp/pull/785

Reply:
That one is merged now.